modules/optimizing-the-MTU-for-your-network.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// networking/network-optimization.adoc
+
+[id='optimizing-the-mtu-for-your-network-{context}']
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network
+interface card (NIC) MTU and the SDN overlay's MTU.
+
+MTUs are now autodetected, so you do not need to adjust MTU settings. When the
+Cluster Network Operator runs, it determines the MTU used by the default route,
+then sets the MTU accordingly. If you want to have a fixed MTU size for your
+workloads, then you must set the MTU explicitly, then set the network
+configuration before installation.
+
+.Procedure
+
+To set the MTU explicitly, complete the following.
+
+. Run:
++
+----
+$ openshift-installer create install-config
+----
+
+. Run the following to generate the required manifests:
++
+----
+$ openshift-installer create manifests
+----
+
+. Edit the `manifests/cluster-network-02-config.yml` file.
+
+. Run the following to use the manifests to create a cluster:
++
+----
+$ openshift-installer create cluster
+----
+
+. Set the following options in the `cluster-network-02-config.yaml` file:
++
+----
+apiVersion: networkoperator.openshift.io/v1
+kind: NetworkConfig
+metadata:
+  creationTimestamp: null
+  name: default
+spec:
+  additionalNetworks: null
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  defaultNetwork:
+    openshiftSDNConfig:
+      mode: Networkpolicy
+    otherConfig: null
+    type: OpenshiftSDN
+  serviceNetwork: 172.30.0.0/16
+status: {}
+----
++
+Under `openshiftSDNConfig`, you can set the mode. `Networkpolicy` is the
```

Comment:
The lines about `openshiftSDNConfig` and `Networkpolicy` could maybe be callouts on the above block?